versions/tasks.py

```diff
@@ -129,10 +139,10 @@ def import_version(
         get_release_date_for_version.delay(version.pk, commit_sha, token=token)
 
     # Load release downloads
-    import_release_downloads.delay(version.pk)
+    import_release_downloads(version.pk)
 
     # Load library-versions
-    import_library_versions.delay(version.name, token=token)
+    import_library_versions(version.name, token=token)
```

Comment:
I switched these 2 functions to be called synchronously for a few reasons.
1. `import_version` is only ever called with `.delay` and I don't think we gain much by running these two functions in parallel.
2. Switching this to run synchronously makes it easier for us to `link` a function after the `task_group` above to ensure the `import_version` commands have completed before the purge runs.